ui/src/containers/SampleViewContainer.js

```diff
@@ -41,8 +41,26 @@ import {
   setAttribute,
 } from '../actions/beamline';
 import { stopBeamlineAction } from '../actions/beamlineActions';
+import { find } from 'lodash';
 
 class SampleViewContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.getControlAvailability = this.getControlAvailability.bind(this);
+  }
+
+  getControlAvailability(name) {
+    const available = find(
+      this.props.uiproperties.sample_view_video_controls.components,
+      {
+        id: name,
+        show: true,
+      },
+    ) || { show: false };
+
+    return available.show;
```

Comment:
I'd move the fallback here: `return available?.show || false`

Reply:
oh, yes thats nice :)